### ISSUE TITLE
With a bad token, resin cli can become entirely broken

### ISSUE THREAD

**Issue by pimterry** [resin-io/resin-cli#443]
As part of reproducing #442, I changed the password for a test account to be purely numeric, and started reproducing and solving the issue. I had an old leftover token for this test account though, and with that in place, it was impossible (before or after that fix) to login or logout or see the help, or do anything with the cli, until the token is manually deleted from the disk. Not clear whether it's because the token has expired or because the account details were changed. Removing the token solves the issue.

```bash
> resin help
ResinRequestError: Request error: Unauthorized

If you need help, don't hesitate in contacting us at:

  GitHub: https://github.com/resin-io/resin-cli/issues/new
  Gitter: https://gitter.im/resin-io/chat
```

Haven't investigated in depth, but I think this is because we check `resin.auth.whoami()` when [logging mixpanel events](https://github.com/resin-io/resin-cli/blob/master/lib/events.coffee#L14), before we try to run any commands, and that explodes.

I've got a backup of the broken token I'm happy to share on demand, if that's useful.

**Comment by pimterry** [comment]
Looks like this is related: https://www.flowdock.com/app/rulemotion/resin-frontend/threads/9aaxkeZiIhX9p07X7d81Gy7FTDc

In that case, Joe's hitting a similar looking issue after recreating the devenv, but with the old token leftover.

**Comment by pimterry** [comment]
Fixed in 6.6.8
